src/sentry/runner/commands/configoptions.py

```diff
@@ -0,0 +1,271 @@
+            else:
+                click.echo(f"Failed to update: {key} = {val} (dry run:{dryrun})")
+
+        keysToDelete = data.get("delete", {})
+        keysToDelete = create_key_value_generator(keysToDelete, "\n", ": ")
+        for key in keysToDelete:
+            click.echo(delete(key, dryrun))
+            click.echo(f"Successfully deleted: {key} (dry run:{dryrun})")
+
+
+@configoptions.command()
+@click.argument("filename", required=True)
+@click.option(
+    "--dryrun",
+    is_flag=True,
+    default=False,
+    required=False,
+    help="Output exactly what changes would be made and in which order.",
+)
+@configuration
+def strict(filename: str, dryrun: bool) -> bool:
+    "Deletes everything not in the uploaded file, and applies all of the changes in the file."
+    import yaml
+
+    from sentry import options
+
+    if dryrun:
+        click.echo("Dryrun flag on. ")
+
+    with open(filename) as stream:
+        configmap_data = yaml.safe_load(stream)
+        data = configmap_data.get("data", {}).get("options-strict.yaml", "")
+
+        kv_generator = create_key_value_generator(data, "\n", ": ")
+        db_keys = (opt.name for opt in options.all())
+        for key in db_keys:
+
+            if not can_change(key):
+                continue
+
+            if key not in kv_generator:
+                success = _delete(key, dryrun)
+                if success:
+                    click.echo(f"Successfully deleted: {key} (dry run:{dryrun})")
+                else:
+                    click.echo(f"Failed to deleted: {key} (dry run:{dryrun})")
+
+        kv_generator = create_key_value_generator(data, "\n", ": ")
+        for line in kv_generator:
+            click.echo(f"line: {line} {type(line)}")
+            key = line[0]
+            val = ""
+            click.echo(f"key: {key} {type(key)}")
+            if len(line) > 1:
+                val = line[1]
+            click.echo(f"val: {val} {type(val)}")
+            click.echo(f"dryrun: {dryrun} {type(dryrun)}")
+            success = _set(key, val, dryrun)
+            if success:
+                click.echo(f"Successfully updated: {key} = {val} (dry run:{dryrun})")
+            else:
+                click.echo(f"Failed to update: {key} = {val} (dry run:{dryrun})")
+
+
+@configoptions.command()
+@click.argument("key", required=True)
+@click.option(
+    "--dryrun",
+    is_flag=True,
+    default=False,
+    required=False,
+    help="Output exactly what changes would be made and in which order.",
+)
+@configuration
+def get(key: str, dryrun: bool = False) -> str:
+    return _get(key, dryrun)
+
+
+def _get(key: str, dryrun: bool = False) -> str:
+    from sentry import options
+    from sentry.options.manager import UnknownOption
+
+    try:
+        opt = options.lookup_key(key)
+        click.echo(f"Fetched Key: {opt.name} ({opt.type}) = {options.get(opt.name)}")
+        return opt
```

Comment:
If we cast opt to a string we wont be able to call opt.name/ opt.flags, is this something that is ok or should we leave it as a sentry.options.store.key type